pkg/datapath/linux/config/config.go

```diff
@@ -907,6 +907,8 @@ func (h *HeaderfileWriter) writeTemplateConfig(fw *bufio.Writer, e datapath.Endp
 		fmt.Fprintf(fw, "#define DIRECT_ROUTING_DEV_IFINDEX %d\n", directRoutingIfIndex)
 		if len(option.Config.GetDevices()) == 1 {
 			fmt.Fprintf(fw, "#define ENABLE_SKIP_FIB 1\n")
+		} else if option.Config.EnableBPFFIBCustomRules {
+			fmt.Fprintf(fw, "#define ENABLE_BPF_FIB_CUSTOM_RULES 1\n")
```

Comment:
I would just define a `FIB_LOOKUP_FLAG` right here, and if option.Config.EnableBPFFIBCustomRules is enabled set this to 0, otherwise "BPF_FIB_LOOKUP_DIRECT" string. Then you don't need the whole ENABLE_BPF_FIB_CUSTOM_RULES detour.

Reply:
good idea!